atom/common/api/atom_bindings.h

```diff
@@ -11,6 +11,9 @@
 #include "base/strings/string16.h"
 #include "v8/include/v8.h"
 #include "vendor/node/deps/uv/include/uv.h"
+#include "atom/common/crash_reporter/crash_reporter.h"
+
+using crash_reporter::CrashReporter;
```

Comment:
>  I don't know the atom style guides

Electron uses https://google.github.io/styleguide/cppguide.html and can be run with `npm run lint-cpp`